### ISSUE TITLE
bower.json references minified JS file

### ISSUE THREAD

**Issue by callumlocke** [Leaflet/Leaflet#3236]
The [bower.json spec](https://github.com/bower/bower.json-spec#main) says you must not include minified files in `main`. (In general bower packages aren't even supposed to contain minified files at all, but this isn't so strict.)

Current bower.json for leaflet:

```json
{
  "name": "leaflet",
  "version": "0.7.3",
  "description": "JavaScript library for mobile-friendly interactive maps",
  "main": [
    "dist/leaflet.js",
    "dist/leaflet.css",
    "dist/leaflet-src.js",
    "dist/images/layers-2x.png",
    "dist/images/layers.png",
    "dist/images/marker-icon-2x.png",
    "dist/images/marker-icon.png",
    "dist/images/marker-shadow.png"
  ],
  "ignore": [
    ".*",
    "CHANGELOG.json",
    "FAQ.md",
    "debug",
    "spec",
    "src",
    "build"
  ]
}
```

**Comment by IvanSanchez** [comment]
This is fixed in master by #2726 - the issue being that the 0.7.3 release was made before that pull.

@mourner: Is a 0.7.4 release (based off the `stable` branch) somewhere in the horizon? A new build could make bower users happy.

**Comment by mourner** [comment]
@IvanSanchez yeah, I think we should do 0.7.4. There are some useful fixes there, need to check if there's anything else worth backporting.
